source/parser.hera

```diff
@@ -2262,7 +2262,8 @@ ComputedPropertyName
     }
 
 Decorator
-  AtAt IdentifierReference Arguments?
+  # Might want to disallow import, super, and return CallExpressions
```

Comment:
You can use decorators inside a class method, for example, so I *think* they're all plausible...